templates/definition/vehicle/evnotify.yaml

```diff
@@ -26,6 +28,12 @@ render: |
   {{- if ne .phases "" }}
   phases: {{ .phases }}
   {{- end }}
+  {{- if .status }}
+  status: {{ .status }}
```

Comment:
Wie stellst du dir vor dass das funktioniert?

Reply:
Ich hätte gedacht, dass ich den status (A..F) und range (in km) übergebe wie im generic car template, in meinem Fall mithilfe des mqtt plugins.. oder ist dies nur im generischen template möglich?

Reply:
so meine ich das auch nicht.. ich muss den Wert doch eh in der evcc.yaml übergeben.. 

also z.B.

 ```
status:
    source: mqtt
    topic: nodered/evnotify/status
range:
    source: mqtt
    topic: nodered/evnotify/autonomy
```

Reply:
Die eigentliche Logik bei evNotify (=custom vehicle) liegt ja hier:

```
  soc:
    source: http
    uri: https://app.evnotify.de/soc?akey={{ .akey }}&token={{ .token }} # evNotify Server + AKEY
    method: GET
    jq: .soc_display

```
Du kannst dir also auch einfach ein custom vehicle selbst bauen ohne Template.